01week/resume/JavaScript/Javascript.js

```diff
@@ -0,0 +1,71 @@
+function sectionSetup(_selectedButton) {
```

Comment:
Interesting...

Any reason you're using a network request instead of managing the visibility of dropdowns and side bars by dealing with styles.

and xhr request to files that are in your project is like taking a plane to your next door neighbor's house.